libopae/pluginmgr.c

```diff
@@ -71,6 +73,42 @@ STATIC opae_api_adapter_table *adapter_list = (void *)0;
 static pthread_mutex_t adapter_list_lock =
 	PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;
 
+
+#define MAX_PLUGINS PLUGIN_SUPPORTED_DEVICES_MAX
+static plugin_cfg plugin_list[MAX_PLUGINS];
+static int plugin_count;
+
+#define CFG_PATHS 4
+static const char *_opae_cfg_files[CFG_PATHS] = {
+	"/etc/opae.cfg",
+	"~/.config/opae/config",
+	"~/.local/opae/config",
+	"~/.local/opae.cfg"
```

Comment:
Why support different filenames?  Can we converge on opae.cfg?

Reply:
Ok